hack/generate/samples/internal/pkg/utils.go

```diff
@@ -105,3 +106,15 @@ func removeAllAnnotationLines(annotations map[string]string, filePaths []string)
 	}
 	return nil
 }
+
+// CleanLocalDependencies will remove tool binaries so the correct versions are used for each plugin version.
+func (ctx SampleContext) CleanLocalDependencies() {
+	log.Infof("cleaning local dependencies")
+	cmd := exec.Command("rm", "-f", "which", "controller-gen")
```

Comment:
Use native functions, ex. [how kubebuilder does it](https://github.com/kubernetes-sigs/kubebuilder/blob/master/test/e2e/utils/test_context.go#L87-L95).

Reply:
Oh. @estroz tks. Good catcher. 
I did notice that the Prepare does it. So, we are fine. We do not need to do that. 
Closing this one.